package.json

```diff
@@ -20,12 +20,13 @@
     "flumelog-level": "^0.0.5",
     "flumelog-memory": "^1.0.2",
     "flumelog-offset": "^3.4.2",
+    "flumeview-level": "^4.0.2",
     "flumeview-reduce": "^1.3.16",
     "statistics": "^3.3.0",
-    "tape": "^4.13.0"
+    "tape": "^5.0.0"
```

Comment:
`<aside>`
oh oh, `tape@5` has some more hard out checks I think around planning?
At least when I put it in `ssb-db` [_more bugs showed up_](https://github.com/ssbc/ssb-db/pull/289)
`</aside>`